test_pytest_html.py

```diff
@@ -198,7 +199,8 @@ def test_resources(self, testdir):
         assert content
         assert content in html
 
-        regex_css_link = '<link href="assets/style.css" rel="stylesheet"'
+        href = os.path.join('assets', 'style.css')
```

Comment:
This isn't a file path, it's a relative URL, so we don't want to use `os.path.join` here.